contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -129,6 +131,7 @@ abstract contract CompoundingValidatorManager is Governable, Pausable {
     struct ValidatorData {
         ValidatorState state; // The state of the validator known to this contract
         uint40 index; // The index of the validator on the beacon chain
+        bool canActivate; // Has validator ever had minimal activation balance
```

Comment:
Should we add a `ACTIVE` validator state rather than introduce a new state flag flag?